modules/govuk/manifests/node/s_backend_lb.pp

```diff
@@ -108,6 +108,8 @@
   }
 
   nginx::config::vhost::proxy { "docs.${app_domain}" :
-    to => ['govuk-developer-documentation-production.s3-website-eu-west-1.amazonaws.com'],
+    to        => ['govuk-developer-documentation-production.s3-website-eu-west-1.amazonaws.com'],
+    protected => false,
+    http_host => 'govuk-developer-documentation-production.s3-website-eu-west-1.amazonaws.com',
```

Comment:
I've added a comment above. I'm open to making it per-environment if that's less surprising though!